source/guide_owncloud.rst

```diff
@@ -0,0 +1,97 @@
+.. highlight:: console
+
+.. sidebar:: Logo
+  
+  .. image:: _static/images/owncloud.png 
+      :align: center
+
+#########
+ownCloud
+#########
+
+ownCloud_ is an open source cloud solution written in PHP and distributed unter the GAPLv3 licence.
+
+ownCloud was initially released in 2010 and is maintained by the ownCloud GmbH.
+
+----
+
+.. note:: For this guide you should be familiar with the basic concepts of 
+
+  * PHP_
+  * MySQL_ 
+  * domains_
+  * cronjobs_
+
+Prerequisites
+=============
+
+We're using PHP_ in the stable version 7.1:
+
+::
+
+ [isabell@stardust ~]$ uberspace tools version show php
+ Using 'PHP' version: '7.1'
+ [isabell@stardust ~]$
+
+.. include:: includes/my-print-defaults.rst
+
+If you want to use your cloud with your own domain you need to setup your domain first:
+
+.. include:: includes/web-domain-list.rst
+
+Installation
+============
+
+``cd`` to your `document root`_, then download and the latest release of the ownCloud and extract it:
```

Comment:
Please tell users where to find the link to the latest release. Lose the 'and' after download.